.github/workflows/notebook-controller-images-updater.yaml

```diff
@@ -99,7 +128,7 @@ jobs:
             --base ${{ env.BRANCH_NAME }}
       env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          pr_title: "[GHA] Update odh and notebook-controller with image ${{ steps.version.outputs.VERSION }}-${{ steps.commit-id.outputs.COMMIT_ID }}"
+          pr_title: "[GHA-${{ github.run_id }}] Update odh and notebook-controller with image ${{ steps.version.outputs.VERSION }}-${{ steps.commit-id.outputs.GIT_HASH }}"
```

Comment:
Perhaps adding a unique `label` on create (to be subsequently used as a search param on `list` later) allows us to not "pollute" the title?

Just a thought..

Reply:
The `github.run_id` is associated with the `kubeflow release pipeline` gha itself, on the otherside the label will be stay static. We needed a mechanism here to identify which one is the PR created from the specific gha run.

Reply:
I think that Andy meant some unique label for each of these PRs. But I don't like that approach much, since the labels will be then present as option labels for this repo, correct? Unless we would be actively removing them from the PR after the merge maybe.

Reply:
Ewww.. I hadn't considered that @jstourac .. great point.. that is/was a bad suggestion